features/cli/cancel_unpaid_orders.feature

```diff
@@ -0,0 +1,9 @@
+@cli
+Feature: Cancel unpaid orders
+    In order to manually cancel only unpaid orders
+    As a developer
+    I need to use command in terminal
+
+    Scenario: Canceling unpaid orders from console
+        When I run command that cancels unpaid orders
+        Then I should see output "Unpaid orders has been canceled" message in terminal
```

Comment:
```suggestion
        Then I should see be informed that unpaid orders have been cancelled
```

Reply:
Additionally, we need to check if these orders have been cancelled _for real_ 😄 For that we need some background (as mentioned above).

Reply:
Additionally, it would be also great to have a few orders with different states and check to really only those that are awaiting payment are cancelled (as this is the goal of this fix :dancer:)